generator/templates/eval.go.tmpl

```diff
@@ -111,6 +111,9 @@ grpcfed.EvalDef({{ .Ctx }}, value, grpcfed.Def[{{ $def.Type }}, *localValueType]
 		}
 		{{- end }}
 		{{- end }}
+		{{- if $def.IsCall }}
+		s.logger.DebugContext(ctx, "call {{ $def.MethodFQDN }}", slog.Any("call_request", s.logvalue_{{ $def.LogValueRequestType }}(args)))
```

Comment:
Here, I want to specify the FQDN of the request message

e.g.) `slog.Any("pkg.FooRequest", s.logvalue_Pkg_FooRequest(args))`